common/script_values/imperia_values.txt

```diff
@@ -832,139 +832,173 @@ imperia_qing_fragile_unity_value = {
 	if = {
 		limit = {
+			# If Qing does NOT have a presence in Xinjiang (Tarim Basin)
 			NOT = {
-				exists = c:XIN
+				any_state = {
+					state_region = s:STATE_TIANSHAN
+					owner = {
+						OR = {
+							this ?= c:CHI
+							is_subject_of = c:CHI
+						}
+					}
+				}
 			}
 		}
-		add = 10
+		add = 5
 	}
 	if = {
 		limit = {
+			# If Qing does NOT have a presence in Mongolia (Urga)
 			NOT = {
-				exists = c:MGL
+				any_state = {
+					state_region = s:STATE_URGA
+					owner = {
+						OR = {
+							this ?= c:CHI
+							is_subject_of = c:CHI
+						}
+					}
+				}
 			}
 		}
-		add = 10
+		add = 5
 	}
 	if = {
 		limit = {
+			# If Qing does NOT have a presence in Eastern provinces (Southern Manchuria)
 			NOT = {
-				exists = c:MCH
+				any_state = {
+					state_region = s:STATE_SOUTHERN_MANCHURIA
+					owner = {
+						OR = {
+							this ?= c:CHI
+							is_subject_of = c:CHI
+						}
+					}
+				}
 			}
 		}
 		add = 10
 	}
 	if = {
 		limit = {
+			# If Qing does NOT have a presence in Gansu (Xibei, Ningxia)
 			NOT = {
-				exists = c:XIB
+				any_state = {
+					state_region = s:STATE_NINGXIA
+					owner = {
+						OR = {
+							this ?= c:CHI
+							is_subject_of = c:CHI
+						}
+					}
+				}
 			}
 		}
 		add = 10
 	}
 	if = {
 		limit = {
+			# If Qing does NOT have a presence in Tibet (Lhasa)
 			NOT = {
-				exists = c:TIB
+				any_state = {
+					state_region = s:STATE_LHASA
+					owner = {
+						OR = {
+							this ?= c:CHI
+							is_subject_of = c:CHI
+						}
+					}
+				}
 			}
 		}
-		add = 10
+		add = 5
 	}
 	if = {
 		limit = {
+			# If Qing does NOT have a presence in Sichuan
 			NOT = {
-				exists = c:SIH
+				any_state = {
+					state_region = s:STATE_SICHUAN
+					owner = {
+						OR = {
+							this ?= c:CHI
+							is_subject_of = c:CHI
+						}
+					}
+				}
 			}
 		}
 		add = 10
 	}
 	if = {
 		limit = {
+			# If Qing does NOT have a presence in Huguang (Hunan)
 			NOT = {
-				exists = c:HNA
+				any_state = {
+					state_region = s:STATE_HUNAN
+					owner = {
+						OR = {
+							this ?= c:CHI
+							is_subject_of = c:CHI
+						}
+					}
+				}
 			}
 		}
 		add = 10
 	}
 	if = {
 		limit = {
+			# If Qing does NOT have a presence in Liangguang (Guangdong)
 			NOT = {
-				exists = c:YUE
+				any_state = {
+					state_region = s:STATE_GUANGDONG
+					owner = {
+						OR = {
+							this ?= c:CHI
+							is_subject_of = c:CHI
+						}
+					}
+				}
 			}
 		}
 		add = 10
 	}
 	if = {
 		limit = {
+			# If Qing does NOT have a presence in Formosa
 			NOT = {
-				exists = c:FRS
+				any_state = {
+					state_region = s:STATE_FORMOSA
+					owner = {
+						OR = {
+							this ?= c:CHI
+							is_subject_of = c:CHI
+						}
+					}
+				}
 			}
 		}
 		add = 5
 	}
 	if = {
 		limit = {
+			# If Qing does NOT have a presence in Korea (Gyeonggi)
 			NOT = {
-				exists = c:KOR
+				any_state = {
+					state_region = s:STATE_SEOUL
+					owner = {
+						OR = {
+							this ?= c:CHI
```

Comment:
ehm.. it is my understanding that ?= does an exist test on the left side, so it checks whether "this" exists and then checks whether it is c:CHI. So if c:CHI doesn't exist it would still throw errors according to my understanding. You just need to check exists = c:CHI the normal way I think.